pyproject.toml

```
@@ -1,8 +1,9 @@
 [tool.poetry]
 name = "django-guid"
-version = "3.0.1"  # Remember to also change __init__.py version
+version = "3.1.0-rc1"  # Remember to also change __init__.py version
```

```suggestion
version = "3.1.0"  # Remember to also change __init__.py version
```

Pre-release has been created [here](https://pypi.org/project/django-guid/3.1.0rc1/).